forest/src/cli/mod.rs

```diff
@@ -143,6 +143,8 @@ pub struct CliOpts {
     pub height: Option<i64>,
     #[structopt(long, help = "Import a snapshot from a local CAR file or url")]
     pub import_snapshot: Option<String>,
+    #[structopt(long)]
+    pub halt_after_import: bool,
```

Comment:
Please add some doc comment help for this new flag. Maybe refactor to use doc comments for the other flags too.

Reply:
@lemmy do you think such self-describing flags need help comments?

Reply:
Yes. The help message could even add more details such as "Halt with exit code 0 after successfully importing a snapshot".